backend/data/scrapers/payloadGened.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
Does the gened payload need its own file?
Once it's finalised, it may be nice to see if it can be incorporated into the current structure in `payload.py`
If they're too different or, combining them is really messy then at worst, we can just rename that file to `payloads.py` and import the regular payloads and gened_payload out of that file.